src/PerfView.Tests/EventViewerTests.cs

```diff
@@ -0,0 +1,311 @@
+using System;
```

Comment:
I was actually hoping that the regression test would exercise the code change in PerfView directly.  Can you replace this new test with one that exercises the code change in the EventViewer?